coalib/collecting/Collectors.py

```diff
@@ -284,13 +285,9 @@ def collect_registered_bears_dirs(entrypoint):
     :param entrypoint: The entrypoint to find packages with.
     :return:           List of bear directories.
     """
-    collected_dirs = []
-    for ep in pkg_resources.iter_entry_points(entrypoint):
-        registered_package = None
-        try:
-            registered_package = ep.load()
-        except pkg_resources.DistributionNotFound:
-            continue
-        collected_dirs.append(os.path.abspath(
-            os.path.dirname(registered_package.__file__)))
-    return collected_dirs
+    eps = {find_loader("bears")
```

Comment:
Are you assuming that the bear package will have a module named "bears" ?
That;s only valid in coala-bears right ?